playground/playground.go

```diff
@@ -62,13 +63,16 @@ func main() {
 
 		packages := make(map[string]*compiler.Archive)
 		var pkgsToLoad map[string]struct{}
-		importContext := compiler.NewImportContext(func(path string) (*compiler.Archive, error) {
-			if pkg, found := packages[path]; found {
-				return pkg, nil
-			}
-			pkgsToLoad[path] = struct{}{}
-			return &compiler.Archive{}, nil
-		})
+		importContext := &compiler.ImportContext{
+			Packages: map[string]*types.Package{"unsafe": types.Unsafe},
```

Comment:
The `unsafe` entry should be unnecessary since my latest commit https://github.com/gopherjs/gopherjs/commit/f8510c9aef19750ec5a1178b33932a7f9defd197